### PR TITLE
Remove flutterNext

### PR DESCRIPTION
**Pull request by jmagman**
This flag's watch has ended.

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by jonahwilliams** [comment]
(I think this is OK for test exempt, since we're inlining a constant - fyi @Hixie )

**Comment by jmagman** [comment]
> (I think this is OK for test exempt, since we're inlining a constant - fyi @Hixie )

The actual resulting `enabledByDefault` and `available` values are fully tested in https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/general.shard/features_test.dart

**Comment by jonahwilliams** [comment]
The, uh, test coverage appears to be working

File: packages/flutter_tools/lib/src/features.dart [hunk 1]
```diff
@@ -102,8 +102,6 @@ const Feature flutterMacOSDesktopFeature = Feature(
   name: 'beta-quality support for desktop on macOS',
   configSetting: 'enable-macos-desktop',
   environmentOverride: 'FLUTTER_MACOS',
-  extraHelpText: flutterNext ?
-      'Newer beta versions are available on the beta channel.' : null,
```
Comment (jonahwilliams) [inline packages/flutter_tools/lib/src/features.dart:106]: Make sure to leave the extra help text for this and the other desktop features

**Comment by Hixie** [comment]
test exempt: deleting code

File: packages/flutter_tools/lib/src/features.dart (hunk 1 above)
Comment (jmagman) [reply packages/flutter_tools/lib/src/features.dart:106]: Hmm, is that still a helpful thing to say though? What does "Newer beta versions are available on the beta channel" mean if you're on the dev channel?  I assume it references "beta-quality" but what does the beta channel have to do with it once it's available on all channels?

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/features.dart:106]: That is the current correct messaging though - even though desktop is available on stable, it is meant to be a "beta preview"

Comment (jmagman) [reply packages/flutter_tools/lib/src/features.dart:106]: But if you're on the `dev` channel then what does `'Newer beta versions are available on the beta channel.` mean?
Anyway it's not really related to this change, I'll revert the `extraHelpText` part.

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/features.dart:106]: Yeah, its certainly awkward. I dunno if we want to let the message be configured per-channel, but that would be one way to do it

**Pull request merged**